.travis.yml

```diff
@@ -32,17 +32,6 @@ jobs:
       <<: *localnet
       script:
         - docker exec dev ./scripts/travis/dex-tests.sh
-    - <<: *defaults
```

Comment:
These are not needed due to https://github.com/project-serum/serum-dex/pull/95.

Removing to speed up CI.